server/src/queries/mod.rs

```diff
@@ -39,5 +88,49 @@ pub async fn get_signals(db: &Pool<Postgres>) -> Result<Vec<DBResponse>, sqlx::E
         AND NOT b.chain_status = 'orphaned'
         AND buc.status = 'applied'
         "#
-    ).fetch_all(db).await
+        ).fetch_all(db).await
+    }
+}
+
+pub async fn build_router(context: ApiContext) -> Router {
+    let cors = CorsLayer::new()
+        .allow_methods([Method::GET, Method::POST])
+        .allow_origin(Any);
+    axum::Router::build_v1(&context.config)
+        .layer(ServiceBuilder::new().layer(cors).layer(Extension(context)))
+}
+
+pub fn create_config() -> Option<Config> {
```

Comment:
As far as I can tell this function is solely used in a testing context - so it should find it's definition there as well.
Most likely - inside ` __mocks__` & renamed to `create_mock_config`